htdocs/api/v0.0.3-dev/Candidates.php

```diff
@@ -132,7 +132,7 @@ public function handlePOST()
                 "CenterID"
             );
             $allUserSiteNames = $this->DB->pselectColWithIndexKey(
-                "SELECT CenterID, Name FROM psc WHERE FIND_IN_SET(CenterID, :cid)",
+                "SELECT CenterID, Name FROM psc WHERE CenterID IN (:cid)",
```

Comment:
Won't work for multisite users

Reply:
Since the $centerIDs variable is coming from data in the DB, we could do 
```... WHERE CenterID IN ". implode(",", $centerIDs ...```
Any objection for that?

Reply:
Using string concatenation to build a query is much less secure and error prone than using a bind parameter.

Reply:
@driusan error prone yes, we would have to check if data is INT of STRING. 
Less secure? When it is a list of INT that where extracted from the DB, risk of injection are nil unless someone can manipulate the code, and in that case, prepared statement can be manipulated anyway or the DB was tempered.

It might not be a 50% overall increase in speed, but the gain of efficiency is not negligible.